File: wrappers/dotnet/indy-sdk-dotnet/PoolApi/PoolLedgerTerminatedException.cs
namespace Hyperledger.Indy.PoolApi
{
    /// <summary>
    /// Exception thrown when the pool ledger was terminated.
    /// </summary>
    public class PoolLedgerTerminatedException : IndyException
    {
        const string message = "The pool ledger was terminated.";

        /// <summary>
        /// Initializes a new PoolLedgerTerminatedException.
        /// </summary>
        internal PoolLedgerTerminatedException() : base(message, (int)ErrorCode.PoolLedgerTerminated)
        {

        }
    }

}

File: wrappers/android/src/main/java/org/hyperledger/indy/sdk/pool/PoolLedgerTerminatedException.java
package org.hyperledger.indy.sdk.pool;

import org.hyperledger.indy.sdk.ErrorCode;
import org.hyperledger.indy.sdk.IndyException;

/**
 * Exception thrown when a pool ledger has been terminated.
 */
public class PoolLedgerTerminatedException extends IndyException
{
	private static final long serialVersionUID = 768482152424714514L;
	private final static String message = "The pool ledger was terminated.";

	/**
	 * Initializes a new PoolLedgerTerminatedException.
	 */
	public PoolLedgerTerminatedException()
	{
		super(message, ErrorCode.PoolLedgerTerminated.value());
	}
}